wd_helper.au3

```diff
@@ -35,7 +35,7 @@
 #cs
 	- Jonathan Bennett and the AutoIt Team
 	- Thorsten Willert, author of FF.au3, which I've used as a model
-	- Michał Lipok (@mLipok) for all his feedback / suggestions
+	- Michał Lipok (@mLipok) for all his feedback / suggestions / contribution

```

Comment:
You are thanking yourself? LOL

Reply:
Now I have tweaked a little for a less extensive description.
But I wonder why I do not see here my recent change (from about few minutes ago)